omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -2308,9 +2326,12 @@ def draw_text(self, text, x, y, fontsize, rgb, align="center"):
 
         temp_label = self.draw_temp_label(text, fontsize, rgb)
 
-        if align == "vertical":
+        if align == "left-vertical":
             temp_label = temp_label.rotate(90, expand=True)
             y = y - (temp_label.size[1]/2)
+        elif align == "right-vertical":
+            temp_label = temp_label.rotate(-90, expand=True)
+            y = y - (temp_label.size[1]/2)
```

Comment:
I think you also need `x = x - temp_label.size[0]` here to shift the labels a bit to the left:

Reply:
OK !